views/partials/data-removal/remove-kanary-info.hbs

```diff
@@ -8,9 +10,10 @@
     />
     <h2 class="remove-kanary-title">Keep Your Personal Information Private With
       Kanary</h2>
-    <p class="remove-txt-cntr">If you are interested in continuing your data
-      removal coverage while we evaluate next steps for our offering, please
-      consider signing up directly with Kanary.</p>
+    <p class="remove-txt-cntr">Firefox Monitor's data removal pilot is coming to an end on April 20, 2022. Thank you for
```

Comment:
probably obvious - but we'll just need to remove this line manually when that date actually comes (the block of text that appears once it is ended should cover the explanation) - and instate the new date for the second group at the appropriate time.

<img width="831" alt="Screen Shot 2022-03-29 at 10 57 07 AM" src="https://user-images.githubusercontent.com/520541/160675136-2c455daa-d55f-4ddf-9610-d5029de6e0bd.png">